app/Models/Resume.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Resume extends Model
+{
+    use HasFactory;
+
+    protected $table = 'developer_cvs';
+
+    protected $fillable = [
```

Comment:
```suggestion
    protected $guarded = []
```
 protected $guarded = []